aten/src/ATen/native/native_functions.yaml

```diff
@@ -5310,6 +5310,11 @@
 
 - func: to_mkldnn_backward(Tensor grad, Tensor input) -> Tensor
 
+- func: quantize_per_tensor_dynamic(Tensor self, ScalarType dtype, bool reduce_range) -> Tensor
+  variants: function
+  dispatch:
+    CPU, CUDA: quantize_per_tensor_dynamic
```

Comment:
Hey @jerryzh168! This PR adds `torch.quantize_per_tensor_dynamic` to the python frontend, but it doesn't have any associated API docs. For the 1.11 release, we want to either remove the op from python, or add some public docs.

If you want to add docs for it, can you make a patch and cherry-pick it into the 1.11 release branch?

Otherwise if you're okay with keeping it out of python, let me know and I can make a patch.

Reply:
sure, I think we can add a doc for this, when is the deadline for cherrypick?

Reply:
@jerryzh168 AFAIK there isn't a deadline yet, but you can track the progress here (or ask questions): https://fb.workplace.com/groups/671272153461300/permalink/999082684013577/